comm/client.go

```diff
@@ -0,0 +1,86 @@
+package comm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"strings"
+
+	"os"
+
+	"github.com/gorilla/http"
+	//"net/http"
+
+	"github.com/phomer/scheduler/accounts"
+	"github.com/phomer/scheduler/datastore"
+	"github.com/phomer/scheduler/log"
+)
+
+func NewClient() http.Client {
+	return http.DefaultClient
+}
+
+func MakeRequest(config *accounts.ClientConfig, request *Request) string {
+
+	url := "http://127.0.0.1:8000/" // TODO: Get this from config
+	client := NewClient()
+
+	buffer := datastore.Serialize(request)
+	reader := strings.NewReader(string(buffer))
+
+	status, args, read_closer, err := client.Post(url, nil, reader)
+	if err != nil {
+		switch value := err.(type) {
+		default:
+			log.Dump("error", err)
+			log.Fatal("Get", err)
+			_ = value
+		}
```

Comment:
I don't get the point of this

Reply:
I had some code in there to switch on the error, to print out nicer messages. I cut it, but left the type switch on the assumption that I'd get back to it.